modules/scripts/startup-script/variables.tf

```diff
@@ -30,6 +30,12 @@ variable "region" {
 }
 
 
+variable "gcs_bucket_full_path" {
```

Comment:
I assume pre-commits are failing because docs need to be updated with this variable. This can be done automatically by pre-commits. I can help you to get set up or do it myself on Monday.

Reply:
resolved